csvinsight/split.py

```diff
@@ -188,7 +188,11 @@ def split_in_memory(reader, list_columns=[], list_separator=LIST_SEPARATOR):
     if six.PY2:
         list_columns = [six.binary_type(col) for col in list_columns]
         list_separator = six.binary_type(list_separator)
-    header = next(reader)
+
+    try:
+        header = next(reader)
+    except StopIteration:
+        raise ValueError('Header may not be None')
```

Comment:
I think this message is misleading. You really mean the reader is empty, not the header, right?